saleor/plugins/checks.py

```diff
@@ -1,58 +0,0 @@
-from typing import TYPE_CHECKING, List
-
-from django.conf import settings
-from django.core.checks import Error, register
-from django.utils.module_loading import import_string
-
-if TYPE_CHECKING:
-    from .base_plugin import BasePlugin
-
-
-@register()
-def check_plugins(app_configs, **kwargs):
```

Comment:
Have you checked the new flow with cases like - missing/incorrect path to plugin, missing manager, incorrect import inside the plugin,